src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageMetadataControl.xaml.cs

```diff
@@ -28,20 +30,23 @@ private void ViewLicense_Click(object sender, RoutedEventArgs e)
         {
             if (DataContext is DetailedPackageMetadata metadata)
             {
+          
                 var window = new LicenseFileWindow()
                 {
                     DataContext = new LicenseFileData
                     {
                         Header = string.Format(CultureInfo.CurrentCulture, UI.Resources.WindowTitle_LicenseFileWindow, metadata.Id),
-                        LicenseContent = UI.Resources.LicenseFile_Loading
+                        LicenseText = new FlowDocument(new Paragraph(new Run(UI.Resources.LicenseFile_Loading)))
                     }
                 };
 
                 NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
                 {
                     var content = metadata.LoadFileAsText(metadata.LicenseMetadata.License);
+                    var flowDoc = new FlowDocument();
```

Comment:
Is it necessary to create a new document? Can we just remove existing paragraphs (or clear the blocks) and add the new ones?